src/index.html

```diff
@@ -2,13 +2,54 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header-logo">
+        <a href="#home">
+          <img src="./images/logo.png" alt="logoMoyo">
+        </a>
+      </div>
+
+      <nav class="header-navigation">
+        <ul class="header-navigation__list">
+          <li class="header-navigation__item">
+            <a href="#apple" class="header-navigation__link is-active">Apple</a>
```

Comment:
1) you can simplify your class names
2) if you have more than 2 attributes write each of them from a new line
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a 
              href="#apple" 
              class="nav__link is-active"
            >
              Apple
            </a>
```